src/ontology/cl-edit.owl

```diff
@@ -8936,11 +8936,17 @@ SubClassOf(obo:CL_0000641 obo:CL_0000163)
 
 # Class: obo:CL_0000642 (folliculostellate cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb") obo:IAO_0000115 obo:CL_0000642 "A supportive cell of the vertebrate pituitary that provides macromolecular transport and secretes hormones.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:18287078") Annotation(oboInOwl:hasDbXref "PMID:34550453") Annotation(oboInOwl:hasDbXref "PMID:34734454") Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") obo:IAO_0000115 obo:CL_0000642 "A supporting cell of the anterior pituitary (adenohypophysis). A folliculostellate cell can proliferate and differentiate into all pituitary endocrine cell types. This cell has been described as a small, agranular, star-shaped cell with long, slender processes.")
```

Comment:
This cell type has a "characteristic stellate morphology", hence "stellate" in the name - consider bringing that into the first sentence of the definition.